runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -354,6 +357,16 @@ public void testBraceletSlaughter()
 
 		assertEquals(30, slayerPlugin.getSlaughterChargeCount());
 
+		chatMessageEvent = new ChatMessage(SERVER, "", BRACLET_SLAUGHTER_V3, null);
```

Comment:
This all seems fine. You could maybe put this logic in a separate test case since the current one is very large and this logic is mostly unrelated to it.